appveyor.yml

```diff
@@ -11,6 +11,14 @@ init:
           Update-AppveyorBuild -Version ($env:APPVEYOR_REPO_TAG_NAME).TrimStart("v")
       }
 
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  #version: '{version}'
+  #package_version: '{version}'
+  #assembly_version: '{version}'
+  #file_version: '{version}'
```

Comment:
I assume this whole section can now got since you are now using `APPVEYOR_BUILD_VERSION`.